### PR TITLE
[stable/heapster] add eventer to heapster chart

### PR DESCRIPTION
**Pull request by smelchior**
This PR adds eventer to the heapster chart. It is part of the heapster container so i figured it is best to add it to the same deployment.
Eventer allows you to ship the Kubernetes event logs to another location.

### CODE REVIEW

File: stable/heapster/values.yaml [hunk 1]
```diff
@@ -81,3 +81,37 @@ rbac:
...
+## it uses the same image as heapster but has its own resizer nanny pod
+## eventer is disabled by default
+## see https://github.com/kubernetes/heapster/blob/master/docs/overview.md for the flags you can use
+## you will probably want to change the --sink parameter
+eventer:
+  enabled: false
+  flags:
+  - "--source=kubernetes:https://kubernetes.default"
+  - "--sink=log"
+  resources:
```
Comment (mgoodness) [inline stable/heapster/values.yaml]: Set `resources: {}` and comment the sub-values.

File: stable/heapster/values.yaml [hunk 2]
```diff
@@ -81,3 +81,37 @@ rbac:
...
+    limits:
+      cpu: 100m
+      memory: 250Mi
+    requests:
+      cpu: 100m
+      memory: 250Mi
+  resizer:
+    enabled: true
+    resources:
```
Comment (mgoodness) [inline stable/heapster/values.yaml]: Set resources: {} and comment the sub-values.

**Review by mgoodness — CHANGES_REQUESTED** [review]

**Comment by smelchior** [comment]
you mean like this?
Could you explain why?

**Comment by mgoodness** [comment]
That's correct. We generally avoid setting default resource requests & limits because it can be difficult for users to override them later.

**Comment by smelchior** [comment]
ok, then we should be ready to go :)

**Pull request merged**